packages/react-hooks/src/useCopyToClipboard.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState, useRef } from 'react';
+import { CopyClipboardUtils } from '@deephaven/utils';
+import Log from '@deephaven/log';
+
+const TIMEOUT_DELAY = 3500;
+const log = Log.module('useCopyToClipboard');
+
+export default function useCopyToClipboard(): [
+  boolean,
+  (value: string) => void
+] {
+  const [copied, setCopied] = useState(false);
+  const copiedTimeout = useRef<ReturnType<typeof setTimeout> | null>(null);
+
+  function copyToClipboard(value: string): void {
+    CopyClipboardUtils.copyToClipboard(value)
+      .then(() => {
+        setCopied(true);
+      })
+      .catch(e => log.error(`Unable to copy ${value}`, e));
```

Comment:
Ahh we're just eating the error here? Hmmm okay...

Reply:
Yep. User feedback is that setCopied is never true in that case.